tasks/sanity-inspect-image.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-image
+spec:
+  description: >-
+    Get manifest data for the source image and its base image to workspace
+  params:
+    - name: SOURCE_IMAGE
+      description: the fully qualified image name
+      type: string
+  results:
+    - description: Base image the source image is built from
+      name: BASE_IMAGE
+  steps:
+  - name: inspect-image
+    image: quay.io/redhat-appstudio/hacbs-test:latest
+    script: |
+      IMAGE_MANIFEST=$(workspaces.sanity-ws.path)/image_manifest.json
+      BASE_IMAGE_MANIFEST=$(workspaces.sanity-ws.path)/base_image_manifest.json
+      IMAGE_RAW_MANIFEST=$(workspaces.sanity-ws.path)/image_raw_manifest.json
+
+      echo "Inspecting manifest for source image $(params.SOURCE_IMAGE)"
+      skopeo inspect --no-tags docker://$(params.SOURCE_IMAGE) > $IMAGE_MANIFEST
+      skopeo inspect --no-tags --raw docker://$(params.SOURCE_IMAGE) > $IMAGE_RAW_MANIFEST
+
+      echo "Getting base image manifest for source image $(params.SOURCE_IMAGE)"
+      BASE_IMAGE="$(jq -r ".annotations.\"org.opencontainers.image.base.name\"" $IMAGE_RAW_MANIFEST)"
+      if [ $BASE_IMAGE == 'null' ]; then
+        echo "Cannot get base image info from 'annotations'"
+        echo "Trying to get base image info from 'Labels'"
+        BASE_IMAGE="$(jq -r ".Labels.\"org.opencontainers.image.base.name\"" $IMAGE_MANIFEST)"
```

Comment:
must be based on digest, eg. `registry.access.redhat.com/ubi8/ubi:latest` will download newest image.

Reply:
Ok, thanks.